src/views/Project/index.jsx

```diff
@@ -190,7 +218,11 @@ export default class Project extends Component {
         const moreGoodFirstNodes = fetchMoreResult.goodFirst.edges;
         const moreMentoredNodes = fetchMoreResult.mentored.edges;
 
-        // return previousResult;
+        pageCursors.bzGoodFirst[JSON.stringify(products)] =
+          fetchMoreResult.goodFirst.pageInfo;
+        pageCursors.bzMentored[JSON.stringify(products)] =
+          fetchMoreResult.mentored.pageInfo;
```

Comment:
Is there a way to do this without shoving the full list in the pageCursors object (i.e., `JSON.stringify(products)`)?

Reply:
I'm not sure. Maybe I could make a unique hash from the products, then shove that?

Reply:
That makes sense to me.

Reply:
I got rid of the JSON.stringify by joining each product into one string, which will help performance. I don't actually think a hash will help here, because based on my knowledge, it is not possible to make a unique hash shorter than the string.